runtime/onert/core/src/compiler/ShapeValidator.cc

```diff
@@ -1133,9 +1133,15 @@ void ShapeValidator::visit(const ir::operation::RmsNorm &node)
   const auto ifm_index{node.getInputs().at(ir::operation::RmsNorm::Input::INPUT)};
   const auto gamma_index{node.getInputs().at(ir::operation::RmsNorm::Input::GAMMA)};
 
-  OP_REQUIRES(operands.at(ifm_index).shape().rank() == 4);
-  OP_REQUIRES(operands.at(ifm_index).shape() == operands.at(ofm_index).shape());
-  OP_REQUIRES(operands.at(gamma_index).shape().rank() == 1);
+  auto ifm_shape = operands.at(ifm_index).shape();
+  auto ofm_shape = operands.at(ofm_index).shape();
+  auto gamma_shape = operands.at(gamma_index).shape();
```

Comment:
(optional) How about adding `const reference`?  

```suggestion
  const auto& ifm_shape = operands.at(ifm_index).shape();
  const auto& ofm_shape = operands.at(ofm_index).shape();
  const auto& gamma_shape = operands.at(gamma_index).shape();
```

Reply:
> (optional) How about adding `const reference`?

Okay, I'll update it :)